hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -270,9 +275,34 @@ public void testCreateTableWithOwner() throws Exception {
       Assert.assertEquals(owner, hmsTable.getOwner());
       Map<String, String> hmsTableParams = hmsTable.getParameters();
       Assert.assertFalse(hmsTableParams.containsKey(TableProperties.HMS_TABLE_OWNER));
+      Assert.assertEquals(
+          PrincipalType.USER.name(), hmsTableParams.get(TableProperties.HMS_TABLE_OWNER_TYPE));
     } finally {
       catalog.dropTable(tableIdent);
     }
+
+    TableIdentifier tableIdent2 = TableIdentifier.of(DB_NAME, "tbl_group_owned");
```

Comment:
The below part of this test is almost identical to the part above except the table properties. Would it make sense to add a helper function that executes the test based on different properties that it gets as a parameter?

Reply:
Yes, good idea.